### PR TITLE
Dhruv/test cases for password badge/TRAH-4160

### PR DESCRIPTION
**Pull request by Dhruv-deriv**
## Changes:

Test cases for `modals/verification-docs-list-modal/verification-docs-list-modal.tsx`

### CODE REVIEW

**Comment by Dhruv-deriv** [comment]
Need to check if more test cases can be added covering different conditions

File: packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/Listitem.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import ListItem from '../ListItem'; // Adjust the import path as necessary
```
Comment (amina-deriv) [inline packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/Listitem.spec.tsx]: what is this comment for?

File: packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/Listitem.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
```
Comment (amina-deriv) [inline packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/Listitem.spec.tsx]: why is this needed?

File: packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/verification-docs-list-modal.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    });
+
+    it('should show the DerivLightUploadPoiIcon and text', () => {
+        (useIsSelectedMT5AccountCreated as jest.Mock).mockReturnValueOnce({ is_selected_MT5_account_created: true });
+        renderComponent({});
+        expect(screen.getByText('DerivLightUploadPoiIcon')).toBeInTheDocument();
+        expect(screen.getByText('Your account needs verification.')).toBeInTheDocument();
+    });
+
+    // it('should show the LabelPairedChevronRightCaptionBoldIcon', () => {
```
Comment (amina-deriv) [inline packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/verification-docs-list-modal.spec.tsx]: why is this commented?

File: packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/verification-docs-list-modal.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+// Adjust the import path as necessary
```
Comment (amina-deriv) [inline packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/verification-docs-list-modal.spec.tsx]: do we need this?

File: packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/verification-docs-list-modal.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+                poi_status: 'verified',
+                poa_status: 'verified',
+                valid_tin: 'verified',
+            },
+        });
+        renderComponent({});
+        expect(screen.getByText('Verify your account')).toBeInTheDocument();
+    });
+
+    it('should show the DerivLightUploadPoiIcon and text', () => {
```
Comment (amina-deriv) [inline packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/verification-docs-list-modal.spec.tsx:84]: @Dhruv-deriv we need to add test cases for different status of identity, tax and address

File: packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/Listitem.spec.tsx (hunk 1 above)
Comment (Dhruv-deriv) [reply packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/Listitem.spec.tsx]: removed was generated by copilot

File: packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/verification-docs-list-modal.spec.tsx (hunk 3 above)
Comment (Dhruv-deriv) [reply packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/verification-docs-list-modal.spec.tsx]: was shifted to ListItems.Spec.tsx

File: packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/Listitem.spec.tsx (hunk 2 above)
Comment (Dhruv-deriv) [reply packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/Listitem.spec.tsx]: cleaned code

File: packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/verification-docs-list-modal.spec.tsx (hunk 4 above)
Comment (Dhruv-deriv) [reply packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/verification-docs-list-modal.spec.tsx]: removed

File: packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/Listitem.spec.tsx [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+        const props = {
+            id: 'identity',
+            text: 'Failed',
+            status: AUTH_STATUS_CODES.REJECTED,
+            route: '/proof_of_identity',
+        };
+        renderComponent(props);
+
+        expect(screen.getByText('Failed')).toBeInTheDocument();
+        s;
```
Comment (amina-deriv) [inline packages/appstore/src/components/modals/verification-docs-list-modal/__tests__/Listitem.spec.tsx]: remove this

**Review by amina-deriv — APPROVED** [review]

**Pull request merged**